models/book.rb

```diff
@@ -0,0 +1,15 @@
+require 'item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state, :publish_date, :archived
+
+  def initialize(publisher, cover_state, publish_date, id: nil, archived: false)

```

Comment:
**[OPTIONAL]** You can create a public instance variable `title`. Like this while listing the `Books` in the console app, you will be able to show the **name of each book**